doc/node.1

```diff
@@ -182,26 +182,34 @@ Specify ICU data load path. (overrides \fBNODE_ICU_DATA\fR)
 \',\'\-separated list of core modules that should print debug information.
 
 .TP
-.BR NODE_PATH =\fIpath\fR[:\fI...\fR]
-\':\'\-separated list of directories prefixed to the module search path.
+.BR NODE_DISABLE_COLORS =\fI1\fR
+When set to 1 colors will not be used in the REPL.
```

Comment:
Nit: While you're in here editing the file anyway, can you add a comma after `1`? (Makes it clear that the variable should be set to `1` and not `1 colors` or something like that.)